postgis/postgis.sql.in

```diff
@@ -3163,6 +3163,17 @@ CREATE OR REPLACE FUNCTION ST_Buffer(geometry,float8,text)
 	   $$
 	LANGUAGE 'sql' IMMUTABLE STRICT;
 
+-- Availability: 2.2.0
+CREATE OR REPLACE FUNCTION ST_MinimumBoundingRadius(geometry, OUT center geometry, OUT radius double precision)
+    AS 'MODULE_PATHNAME', 'ST_MinimumBoundingRadius'
+    LANGUAGE 'c' IMMUTABLE STRICT;
+
+-- Availability: 1.4.0
+CREATE OR REPLACE FUNCTION ST_MinimumBoundingCircle(geometry, segs_per_quarter integer DEFAULT 48)
```

Comment:
Please do not change the name of the first argument, or upgrades won't work very easily here.